client/app/lib/components/redux-form/SingleFileInput.jsx

```diff
@@ -63,13 +88,17 @@ const translations = defineMessages({
 // TODO: Use the input element as a controller component - https://reactjs.org/docs/forms.html
 class SingleFileInput extends React.Component {
   static propTypes = {
+    meta: PropTypes.object,
     value: PropTypes.shape({
       url: PropTypes.string,
       name: PropTypes.string,
     }),
     input: PropTypes.shape({
       onChange: PropTypes.func.isRequired,
     }),
+    isNotBadge: PropTypes.bool, // to signify non-badge display
```

Comment:
Can add what if no value was provided to this prop - what would be the default value?

Reply:
Default is no value, so it will take the badge view. Developers don't have to explicitly pass in this prop to state that it is a badge. Do you want me to add a comment for that?

Reply:
Can do that at the top of the file :+1

Reply:
Ok

Reply:
Added to the top of the file